etl/steps/data/garden/ggdc/2022-11-28/penn_world_table_national_accounts.countries.json

```diff
@@ -0,0 +1,218 @@
+{
```

Comment:
If I understand correctly, you can delete this file, since you got the country names by mapping ISO codes to harmonized country names.

Reply:
Right, it's because I needed to run the harmonization command to make the code work (because I would also need to delete that part of the code)